python/pyspark/sql/functions.py

```diff
@@ -793,12 +793,12 @@ def ntile(n):
 # ---------------------- Date/Timestamp functions ------------------------------
 
 @since(1.5)
-def current_date():
+def current_date(timeZone=None):
```

Comment:
Would the change beg a different `@since`? It's no longer true that it existed `since(1.5)`, is it? Just asking...no idea how it really should be.

Reply:
Honestly I have no idea too.. Do you know who can we ask to? Thanks.